x/auth/ante/ante.go

```diff
@@ -34,20 +36,20 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, errorsmod.Wrap(sdkerrors.ErrLogic, "bank keeper is required for ante builder")
 	}
 
-	if options.SignModeHandler == nil {
+	if options.SignModeHandler == nil { // TODO HV2: what is the signing mode for heimdall? Is it `SignDoc` ?
 		return nil, errorsmod.Wrap(sdkerrors.ErrLogic, "sign mode handler is required for ante builder")
 	}
 
 	anteDecorators := []sdk.AnteDecorator{
 		NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
 		NewExtensionOptionsDecorator(options.ExtensionOptionChecker),
 		NewValidateBasicDecorator(),
-		NewTxTimeoutHeightDecorator(),
-		NewValidateMemoDecorator(options.AccountKeeper),
-		NewConsumeGasForTxSizeDecorator(options.AccountKeeper),
-		NewDeductFeeDecorator(options.AccountKeeper, options.BankKeeper, options.FeegrantKeeper, options.TxFeeChecker),
+		// NewTxTimeoutHeightDecorator(), // TODO HV2: this is not present in heimdall. Is it safe to remove?
```

Comment:
Are these changes also supposed to be done when creating the simapp's `AnteDecorator`?

Reply:
My question is just that we're doing changes to the set of decorators here, for "production app", but no changes on simApp. Is Heimdal using simApp?

Reply:
Oh, for that I have this comment https://github.com/0xPolygon/cosmos-sdk/blob/mardizzone/POS-1956-auth/simapp/ante.go#L34 
Will be done once simapp is taken care of

Reply:
Resolved